geospaas_adas_viewer/tests.py

```diff
@@ -0,0 +1,125 @@
+from django.test import Client, TestCase
+from django.utils import timezone
+
+from geospaas.base_viewer.tests import BaseViewerHTMLParser
+from geospaas.catalog.models import Dataset
+from geospaas.vocabularies.models import Parameter
+from geospaas_adas_viewer.views import adasIndexView
```

Comment:
Please also fix below
```suggestion
from geospaas_adas_viewer.views import AdasIndexView
```